### DISCUSSION TITLE
Can Anyone Suggest a Windows Alternative?

### DISCUSSION THREAD

**Discussion by Foxxsayy** [kyleneideck/BackgroundMusic#700] [Q&A]
Hi! I found this when looking for a mute.fm alternative (because it seems to be depreciated) - can anyone recommend a Windows alternative that does what Background music does for Mac?

**Comment by Seulasslintan** [comment]
I suggest using EarTrumpet. I believe you can download it from the Microsoft Store. Or search for it, download an install.

**Reply by Foxxsayy** [reply]
Thanks. I have that, but it only changes where music plays. Unfortunately it doesn't pause/play music automatically.
